fuzzing/fuzz.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Generally you should use this ;-)
```suggestion
#!/usr/bin/env bash
```
https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability